### PR TITLE
Add color parser and add Windows build runner

### PR DESCRIPTION
**Pull request by codeblitz97**
In this pull request, I've brought forth two new additions:

- **Windows Build Batch File**: Recognizing that numerous contributors might utilize Windows as their primary operating system.
- **Color Parser**: This parser now handles string hexadecimal to numerical hexadecimal conversion, alongside support for RGB and RGBA formats. Additionally, it includes a `Color` enum to enrich user interaction.

### CODE REVIEW

**Comment by Didas-git** [comment]
Personally i think the color helpers should be moved to the `helpers` package instead of `jsx` and we could even add some more colors to the enum (eg: https://github.com/Didas-git/tasai/blob/3320d3dcf800babf0b9e3abda9173e46cdcd5602/src/structures/color.ts#L367) 

About the windows build file, i dont think its really necessary since bun recently added the `--filter` option, we just beed to "port" the build file to a package.json script to run with bun.

File: packages/test/src/commands/coloredEmbed.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+export default {
+    post: "GLOBAL",
+    data: { name: "colored_embed", description: "Send colored embed" },
+    run: async (interaction) => {
+        await interaction.deferReply();
+
+        const { ws, rest } = await interaction.client.ping();
+
+        const embed = Embed({ title: "Ping", description: `🏓 WebSocket: \`${ws}ms\` | Rest: \`${rest}ms\``, color: Color.SpringGreen });
```
Comment (xHyroM) [inline packages/test/src/commands/coloredEmbed.ts:15]: What about renaming this file to `.tsx` and use it as components?

**Review by xHyroM — CHANGES_REQUESTED** [review]

File: packages/test/src/commands/coloredEmbed.ts (hunk 1 above)
Comment (codeblitz97) [reply packages/test/src/commands/coloredEmbed.ts:15]: Sureee

**Comment by codeblitz97** [comment]
@Didas-git Okay, I understand, I'll move it to the `helpers` package instead of `jsx` and add all those colors as well

**Comment by Didas-git** [comment]
> @Didas-git Okay, I understand, I'll move it to the `helpers` package instead of `jsx` and add all those colors as well

I was also discussing with the contributors (i will also open a discussion here on github about it) and that instead we could redirect people to just use Tasai with lilybird since Tasai has an extremely powerful Color class and even other helpers that can help both with logging and coloring messages with `ascii` for code blocks.

**Comment by codeblitz97** [comment]
> > @Didas-git Okay, I understand, I'll move it to the `helpers` package instead of `jsx` and add all those colors as well
> 
> I was also discussing with the contributors (i will also open a discussion here on github about it) and that instead we could redirect people to just use Tasai with lilybird since Tasai has an extremely powerful Color class and even other helpers that can help both with logging and coloring messages with `ascii` for code blocks.

Yes, I can already see that, but just in case add it for the new developers who probably aren't familiar with Tasai

**Pull request closed without merging**